flexmeasures/data/services/scheduling.py

```diff
@@ -249,7 +290,8 @@ def make_schedule(
     if scheduler_specs:
         scheduler_class: Type[Scheduler] = load_custom_scheduler(scheduler_specs)
     else:
-        scheduler_class: Type[Scheduler] = find_scheduler_class(sensor)
+        entity = sensor if asset is None else asset
+        scheduler_class: Type[Scheduler] = find_scheduler_class(entity)
```

Comment:
To avoid repetition, we could also do `find_scheduler_class(sensor, asset)` and let that method decide which is prevalent in a single place.

Reply:
With the introduction of the `asset_or_sensor` variable, we are disregarding the idea of having the `asset` as an input variable. This means that we don't do the splitting.